<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>820</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>768</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
            <item>
             <widget class="QGroupBox" name="gbAudio">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>430</width>
                <height>100</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>88</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="title">
               <string/>
              </property>
              <widget class="QPushButton" name="btnAudio1">
               <property name="geometry">
                <rect>
                 <x>150</x>
                 <y>40</y>
                 <width>55</width>
                 <height>55</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QPushButton" name="btnAudio2">
               <property name="geometry">
                <rect>
                 <x>220</x>
                 <y>40</y>
                 <width>55</width>
                 <height>55</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QPushButton" name="btnAudio3">
               <property name="geometry">
                <rect>
                 <x>290</x>
                 <y>40</y>
                 <width>55</width>
                 <height>55</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QPushButton" name="btnAudio4">
               <property name="geometry">
                <rect>
                 <x>360</x>
                 <y>40</y>
                 <width>55</width>
                 <height>55</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="lbAudioSwitch">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>35</y>
                 <width>141</width>
                 <height>40</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>聲道選擇 ：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
              <widget class="QLabel" name="lbAudio1">
               <property name="geometry">
                <rect>
                 <x>150</x>
                 <y>10</y>
                 <width>55</width>
                 <height>25</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="text">
                <string>現場</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
              <widget class="QLabel" name="lbAudio2">
               <property name="geometry">
                <rect>
                 <x>220</x>
                 <y>10</y>
                 <width>55</width>
                 <height>25</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="text">
                <string>葡文</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
              <widget class="QLabel" name="lbAudio3">
               <property name="geometry">
                <rect>
                 <x>290</x>
                 <y>10</y>
                 <width>55</width>
                 <height>25</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="text">
                <string>中文</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
              <widget class="QLabel" name="lbAudio4">
               <property name="geometry">
                <rect>
                 <x>360</x>
                 <y>10</y>
                 <width>55</width>
                 <height>25</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="text">
                <string>其他</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="gbInfo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>350</width>
                <height>100</height>
               </size>
              </property>
              <property name="title">
               <string/>
              </property>
              <widget class="QLabel" name="label">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>20</y>
                 <width>120</width>
                 <height>30</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="text">
                <string>開始時間 ：</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_2">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>60</y>
                 <width>120</width>
                 <height>30</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="text">
                <string>目前時間 ：</string>
               </property>
              </widget>
              <widget class="QLabel" name="lbRecordTime">
               <property name="geometry">
                <rect>
                 <x>130</x>
                 <y>20</y>
                 <width>240</width>
                 <height>30</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>2018-01-30 15:20:10</string>
               </property>
              </widget>
              <widget class="QLabel" name="lbCurTime">
               <property name="geometry">
                <rect>
                 <x>130</x>
                 <y>60</y>
                 <width>240</width>
                 <height>30</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>17:54:20</string>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QFrame" name="frame_3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
                  <item>
                   <widget class="QPushButton" name="btnFb">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>30</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>30</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSlider" name="sliProcess">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnFf">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>30</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>30</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_2">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <property name="lineWidth">
                  <number>0</number>
                 </property>
                 <widget class="QPushButton" name="btnOpen">
                  <property name="geometry">
                   <rect>
                    <x>90</x>
                    <y>20</y>
                    <width>30</width>
                    <height>30</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>O</string>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="btnPlay">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>0</y>
                    <width>50</width>
                    <height>50</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="btnStop">
                  <property name="geometry">
                   <rect>
                    <x>60</x>
                    <y>20</y>
                    <width>30</width>
                    <height>30</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="btnIdle">
                  <property name="geometry">
                   <rect>
                    <x>90</x>
                    <y>20</y>
                    <width>30</width>
                    <height>30</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>I</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="lbProcess">
                  <property name="geometry">
                   <rect>
                    <x>389</x>
                    <y>20</y>
                    <width>131</width>
                    <height>20</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>00:00/00:00</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="btnPause">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>0</y>
                    <width>50</width>
                    <height>50</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                 <widget class="QSlider" name="sliVolume">
                  <property name="geometry">
                   <rect>
                    <x>200</x>
                    <y>20</y>
                    <width>180</width>
                    <height>20</height>
                   </rect>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                 <widget class="QLabel" name="lbVolume">
                  <property name="geometry">
                   <rect>
                    <x>160</x>
                    <y>15</y>
                    <width>30</width>
                    <height>30</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="btnPre">
                  <property name="geometry">
                   <rect>
                    <x>90</x>
                    <y>20</y>
                    <width>30</width>
                    <height>30</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="btnNext">
                  <property name="geometry">
                   <rect>
                    <x>120</x>
                    <y>20</y>
                    <width>30</width>
                    <height>30</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="loVideoList"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="loSinVideo"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
